tests/test_kiwi.py

```diff
@@ -50,17 +49,6 @@ def test_kiwi_installation(auto_container):
     )
 
 
-# Disabling this test with `podman` as it mounts the newly created iso image (by kiwi)
-# on a `/dev/loop*` device, which needs a rootful container for `root:disk` ownership.
-# In rootless mode, `/dev/loop*` devices are owned by `nobody:nobody`,
-# causing `losetup -f --show /tmp/myimage/kiwi-test-image-disk.x86_64-1.15.3.raw` to fail with "Permission denied".
-#
-# Also ref: https://github.com/containers/podman/issues/17715#issuecomment-1460227771
-# Mounting a loop device in rootless mode is not allowed by the kernel.
-@pytest.mark.skipif(
-    not DOCKER_SELECTED,
```

Comment:
thanks, but this still breaks for non-root podman. can we add a skipif for the case of nonroot podman? maybe the assert on the loop device permission needs to be turned into a skip to achieve that?

Reply:
@dirkmueller, added a skipif for non-root podman case.
 
 Question - [here ](https://github.com/SUSE/BCI-tests/blob/main/.github/workflows/ci.yaml#L358-L364) is the only place where I understood tests are run with podman (in this case, with `sudo`). How/Where are the rootless podman tests triggered?

Reply:
not in CI, but in manual execution (most BCI developers use rootless podman and run tests locally)